modules/identity-provider-htpasswd-update-users.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-htpasswd-identity-provider.adoc
+
+[id="identity-provider-htpasswd-update-users_{context}"]
+= Updating users for an HTPasswd identity provider
+
+You can add or remove users from an existing HTPasswd identity provider.
+
+.Prerequisites
+
+* You have created a secret that contains the HTPasswd user file. This procedure assumes that it is named `htpass-secret`.
+* You have configured an HTPasswd identity provider. This procedure assumes that it is named `my_htpasswd_provider`.
+* You have access to the `htpasswd` utility. On Red Hat Enterprise Linux this is available by installing the `httpd-tools` package.
+* You have cluster administrator privileges.
+
+.Procedure
+
+. Retrieve the HTPasswd file from the `htpass-secret` secret and save the file to your file system:
++
+----
+$ oc get secret htpass-secret -ojsonpath={.data.htpasswd} -n openshift-config | base64 -d > users.htpasswd
+----
+
+. Add or remove users from the `users.htpasswd` file.
+
+** To add a new user:
++
+----
+$ htpasswd -b users.htpasswd <username> <password>
+Adding password for user <username>
+----
+
+** To remove an existing user:
++
+----
+$ htpasswd -D users.htpasswd <username>
+Deleting password for user <username>
+----
```

Comment:
why are we documenting how to work with `htpasswd`? That's what `man` is for

Reply:
@stlaz We did it in the existing docs [1], so I just continued with that in the procedure here. Would you prefer that I make this step more generic (and just say to add or remove users from the htpasswd file)?

[1] https://docs.openshift.com/container-platform/4.3/authentication/identity_providers/configuring-htpasswd-identity-provider.html#identity-provider-creating-htpasswd-file-linux_configuring-htpasswd-identity-provider

Reply:
I guess the htpasswd options will hardly change, and if we already have a mention there, let's keep this, then